packages/storybook-test/stories/link/link.stories.tsx

```diff
@@ -146,3 +230,129 @@ export const ExtremelyLongLink: Story = {
     status: { type: [] },
   },
 };
+
+export const Hover: Story = {
+  name: 'Link:hover',
+  args: {
+    children: 'voorbeeldsite (opent in een nieuw venster)',
+    className: 'nl-link--hover',
+    href: 'https://example.com',
+    target: '_blank',
```

Comment:
Die opent in nieuw venster stories moeten we niet de default maken, het is juist een anti-pattern.